cluster/gce/util.sh

```diff
@@ -869,6 +853,30 @@ function kube-push {
   echo
 }
 
+function kube-update-nodes() {
```

Comment:
nit to make me happy: having "Assumed vars:" section here would be awesome, given the extensive prereqs this function has and that it's called from multiple places here.